ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -216,6 +216,9 @@
                     value: ["{{ content_provider_registry_ip }}:5001"]
               {% endif %}
 
+                  - op: add
+                    path: /spec/nodeTemplate/ansible/ansibleVars/edpm_sshd_allowed_ranges
+                    value: ["{{ crc_ci_bootstrap_networks_out['compute-0'].default.ip | ansible.utils.ipaddr('network/prefix') }}"]
```

Comment:
so we also need to accept connections from the zuul executor - else it will fail with a POST_FAILURE or RETRY_LIMIT.

I remember having to do some hack in late tripleo related to this, not how we got out of the situation though... It was about a year and a half ago -.-'.

Reply:
Dang - that's old..
https://opendev.org/openstack/tripleo-quickstart-extras/src/branch/master/roles/overcloud-deploy/tasks/create-scripts.yml#L28-L51

There, we allowed SSH from all, mostly because we wanted to allow access to the nodes in case of debugging.

AFAIK, for now, EDPM is in the same case - allowing from all, correct? Maybe we should stick to that behavior... ?

Reply:
So, I guess we can just do `0.0.0.0/0` then